Makefile

```diff
@@ -32,11 +32,16 @@ SOURCES = \
 CMD_SERVER=bin/server$(EXE)
 CMD_DATA=bin/mkdata$(EXE)
 
+
+RELEASEVERSION ?=$(shell git describe --tags --abbrev=0 | head -n 1)
+# just casually replacing all space characters with non breaking space characters because of reasons...
+COMMITHASH ?=$(shell git log -1 --pretty='format:%h: %s - %an' | sed 's/ / /g')
```

Comment:
Hmm, idk if I like this.  Would it be possible to just escape the spaces instead?